File: CS50x/Week-6/sentimental-mario-more/mario.py
from cs50 import get_int

answer = 0
sharp = 1

while True:
    answer = get_int("Height: ")
    if answer in range(1, 9):
        break

space = answer - 1

for i in range(answer):

    for j in range(space):
        print(" ", end="")

    space -= 1

    for l in range(sharp):
        print("#", end="")

    print("  ", end="")

    for o in range(sharp):
        print("#", end="")

    sharp += 1

    print("")

File: CS50x/Week-1/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int size;
    while (true)
    {
        size = get_int("Pyramid size(1-8): ");
        if (size <= 8 && size >= 1)
        {
            break;
        }
    }
    int s = size - 1;
    int hash = 1;
    for (int i = 0; i < size; i++)
    {
        for (int j = 0; j < s; j++)
        {
            printf(" ");
        }
        s--;
        for (int l = 0; l < hash; l++)
        {
            printf("#");
        }
        hash++;
        printf("\n");
    }

}